### DISCUSSION TITLE
Add support for rendering circular diagrams

### DISCUSSION THREAD

**Discussion by str8edgedave** [terrastruct/d2#257] [Feature Requests]
Add support for a circular diagram to show a process that does not have a start or end point.

See https://www.conceptdraw.com/How-To-Guide/circular-diagram for example.

**Comment by str8edgedave** [comment]
For example, the following code generates a long vertical chain, with an arrow pointing back to the top.

$ cat test.d2
step-1
step-1 -> step-2
step-2 -> step-3
step-3 -> step-4
step-4 -> step-5
step-5 -> step-1

I realize that D2 can use different rendering engines, so I'm not sure how possible this request is...

**Comment by alixander** [comment]
hey @str8edgedave thanks for the request, it looks like a feature request for D2 so I've moved it to another repo (the text-to-diagram one is specifically for comparisons)
